tests/test_populate_catalog.py

```diff
@@ -269,15 +265,23 @@ def should_update_root_catalog_with_new_version_catalog(subtests: SubTests) -> N
             mode="rb",
         ) as updated_root_metadata_file:
             catalog_json = load(updated_root_metadata_file)
-            assert catalog_json[STAC_LINKS_KEY] == expected_root_catalog_links
+            updated_catalog_json = []
+            for i in catalog_json[STAC_LINKS_KEY]:
+                del i["title"]
```

Comment:
Rather than deleting the title from the actual state you could simply remove the non-existent title from the expected state. That should result in less code overall, and very slightly more coverage.

Reply:
Do you mean something like this https://github.com/linz/geostore/commit/bb4fd6295f1b8d88c724874e82bd5b198bf2d37e?
It was my initial attempt, but it didn't work. Maybe I have missed something? 

`del` didn't work, had to change it to `pop`

Reply:
Something like bb4fd62, except some of the titles should still be there.